fineract-provider/src/test/java/org/apache/fineract/cob/service/InlineLoanCOBExecutorServiceImplTest.java

```diff
@@ -45,17 +52,24 @@ class InlineLoanCOBExecutorServiceImplTest {
     private InlineLoanCOBExecutorServiceImpl testObj;
     @Mock
     private TransactionTemplate transactionTemplate;
-
     @Mock
     private InlineLoanCOBExecutionDataParser dataParser;
+    @Mock
+    private LoanRepository loanRepository;
 
     @Test
     void shouldExceptionThrownIfLoanIsAlreadyLocked() {
         JsonCommand command = mock(JsonCommand.class);
         ThreadLocalContextUtil.setTenant(new FineractPlatformTenant(1L, "default", "Default", "Asia/Kolkata", null));
+        HashMap<BusinessDateType, LocalDate> businessDates = new HashMap<>();
+        LocalDate businessDate = LocalDate.now(ZoneId.systemDefault());
+        businessDates.put(BusinessDateType.BUSINESS_DATE, businessDate);
+        businessDates.put(BusinessDateType.COB_DATE, businessDate.plusDays(1));
```

Comment:
This is wrong. COB date shall be behind by a day of the (default) business date!!